nixos/modules/services/mail/mailman.nix

```diff
@@ -33,6 +33,10 @@ let
       ENGINE = "haystack.backends.whoosh_backend.WhooshEngine";
       PATH = "/var/lib/mailman-web/fulltext-index";
     };
+  } // lib.optionalAttrs cfg.enablePostfix {
+    EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend";
+    EMAIL_HOST = "127.0.0.1";
+    EMAIL_PORT = 25;
```

Comment:
Are you sure that hardcoding port 25 is a good idea here? I could see a case being made for 587 instead.

Reply:
Our postfix (https://gitea.c3d2.de/c3d2/nix-config/src/branch/master/hosts/mail/default.nix) with simple-nixos-mailserver and those smtpd_relay_restrictions https://gitlab.com/simple-nixos-mailserver/nixos-mailserver/-/blob/master/mail-server/postfix.nix#L254-256

```nix
        smtpd_relay_restrictions = [
          "permit_mynetworks" "permit_sasl_authenticated" "reject_unauth_destination"
        ];
```

Only accepts relay mails on smtp server port 25. We tried port 587 with sendmail from an internal VM and that didn't work.

Reply:
Not everyone has your postfix configuration, though.